app/services/data/get-last-claim-children.js

```diff
@@ -0,0 +1,6 @@
+const config = require('../../../knexfile').extweb
+const knex = require('knex')(config)
+
+module.exports = function (reference, eligibiltyId) {
+  return knex.raw(`SELECT * FROM [IntSchema].[getLastClaimChildren] (?, ?)`, [ reference, eligibiltyId ])
```

Comment:
Nothing wrong functionally, just a question regarding style.  Is it good practice to have raw SQL for a simple query?